PerimeterXModule/Internals/Validators/PXCookieValidator.cs

```diff
@@ -1,33 +1,69 @@
 ﻿using System;
 using System.Diagnostics;
+using System.Text.RegularExpressions;
 
 namespace PerimeterX
 {
-	class PXCookieValidator : IPXCookieValidator
+	public class PXCookieValidator : IPXCookieValidator
 	{
-		private PxModuleConfigurationSection config;
+		public static readonly string CALL_REASON_INVALID_COOKIE = "invalid_cookie";
+		public static readonly string CALL_REASON_DECRYPTION_FAILED = "cookie_decryption_failed";
+		public static readonly string CALL_REASON_VALIDATION_FAILED = "cookie_validation_failed";
+		public static string CALL_REASON_NO_COOKIE = "no_cookie";
+		public static string CALL_REASON_EXPIRED_COOKIE = "cookie_expired";
+		public static string CALL_REASON_SENSITIVE_ROUTE = "sensitive_route";
+		public static string CALL_REASON_MOBILE_ERROR = "mobile_error_{0}";
+		public IPXCookieValidator PXOriginalCookieValidator { get; set; }
```

Comment:
If you follow my other comment ([this](https://github.com/PerimeterX/perimeterx-asp-net/pull/31/files#r165111972)), then you can change to `private set`.

Reply:
Assigning `PXOriginalTokenValidator` into a member called `PXOriginalCookieValidator` is somewhat confusing. Why not just call it what it is - `PXOriginalTokenValidator`?

Reply:
please clarify